frontend/src/types/api/queryBuilder/queryBuilderData.ts

```diff
@@ -20,7 +20,7 @@ export interface TagFilterItem {
 	id: string;
 	key?: BaseAutocompleteData;
 	op: string;
-	value: string[] | string;
+	value: string[] | string | number[] | number | boolean[] | boolean;
```

Comment:
do we have such inconsistency ?

Reply:
As of now we have `string[] | string | number | boolean`, `number[]` and `boolean[]` was added by mistake,  I will remove it.